src/css/component.site-header.css

```diff
@@ -0,0 +1,107 @@
+/* ---------- COMPONENT site header ---------- */
+
+.site-header {
+  --site-header-gutters-y: var(--size-6);
+  --site-header-menu-gutters-x: var(--size-3);
+  background: var(--color-white);
+  border-bottom: var(--size-px) solid var(--color-grey-500);
+  box-shadow: 0 1rem 0.625rem -0.5rem hsla(var(--color-black-hsl), 0.05);
+  padding-bottom: var(--site-header-gutters-y);
+  padding-top: var(--site-header-gutters-y);
+  position: sticky;
+  top: calc(-1 * var(--size-px));
+  transition: background-color var(--animation-easing) var(--animation-duration-fast),
+    border var(--animation-easing) var(--animation-duration-fast), box-shadow var(--animation-easing) var(--animation-duration-fast);
+}
+
+.site-header__logo {
+  position: relative;
+  width: var(--size-32);
+  z-index: 12;
+}
+
+.site-header__logo svg {
+  height: auto;
+  max-width: 100%;
+  width: 100%;
+}
+
+/* ---------- home ---------- */
+.home .site-header {
+  background: transparent;
+  border: 0;
+  box-shadow: none;
+}
+
+.home .site-header--stuck {
+  background: var(--color-white);
+  border-bottom: var(--size-px) solid var(--color-grey-500);
+  box-shadow: 0 1rem 0.625rem -0.5rem hsla(var(--color-black-hsl), 0.05);
+}
+
+.home .site-header__logo .logo-ceph__text path {
+  fill: #e9eff6; /* TBC */
+}
+
+.home .site-header__menu-open svg g {
+  stroke: #e9eff6; /* TBC */
+}
+
+.home .site-header--stuck .site-header__logo .logo-ceph__text path,
+.home .site-header--opened .site-header__logo .logo-ceph__text path {
+  fill: #0a0c38; /* TBC */
+}
+
+.home .site-header--stuck .site-header__menu-open svg g,
+.home .site-header--opened .site-header__menu-open svg g {
+  stroke: var(--color-black);
+}
+
+/* ---------- media queries ---------- */
+@media (--mq-sm) {
+  .site-header {
+    --site-header-menu-gutters-x: var(--size-5);
+  }
+}
+
+@media (--mq-md) {
+  .site-header {
+    --site-header-menu-gutters-x: var(--size-7);
+  }
+}
+
+@media (--mq-to-lg) {
+  .site-header__menu {
```

Comment:
Thought: Could this do with `overflow-y: auto` (and are there scrolling implications for how the nav list items might scroll beneath the Ceph logo)? Thinking how this panel's contents will increase with language selector in future.

Reply:
I think we will need an `overflow-y: auto` as based on the latest design of the language picker it could add extra height which could push it beyond the device viewport height, so yeah we'd need to scroll.